draft-ietf-quic-manageability.md

```diff
@@ -109,6 +109,12 @@ recommendations as to which practices should or should not be applied; for each
 practice, it describes what is and is not possible with the QUIC transport
 protocol as defined.
 
+This document focuses solely on network management practices that interact
```

Comment:
s/interact/observe/   better?   Otherwise ok.

Reply:
Good point. Making this change. thanks!